app/scripts/acs/acs-graphs.js

```diff
@@ -106,14 +106,14 @@
                 var chart = nv.models.pieChart()
                     .x(function(d) { return addElipses(d.label, 10); })
                     .y(function(d) { return d.value; })
-                    .tooltipContent(function(d) {
-                        var percent = $filter('number')(d.data.value / total * 100, 1);
-                        return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
-                                ' (', percent, '%)'].join('');
-                    })
                     .color(function () { return '#bc5405'; })
                     .margin({top: 0, left: 10, bottom: 0, right: 10})
                     .showLegend(false);
+                chart.tooltip.contentGenerator(function (d) {
+                    var percent = $filter('number')(d.data.value / total * 100, 1);
+                    return ['<b>', d.data.label, ': </b>', numberFilter(d.data.value),
+                            ' (', percent, '%)'].join('');
+                });
```

Comment:
Here it is again.  Now it seems like a helper function is in order (moving the one above outside the conditional still makes sense, even though it'll be shorter).